acquia/acquia_cms
# ACMS-594: Create Install Wizard for Tour Dashboard

**Motivation**

ACMS-594

**Proposed changes**

Create feature installation wizard for tour dashboard.

**Testing steps**

Visit `/admin/tour/dashboard`, click on Get started button
Verify that wizard is working fine.
Verify that saving individual form is working fine.
**Merge requirements**
- [_Enhancement_] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
